deps/readline.lua

```diff
@@ -108,7 +108,7 @@ function Editor:insertAbove(line)
   end)
 end
 function Editor:insert(character)
-  local display = self.cover or character
+  local display = string.rep(self.cover, #character)
```

Comment:
This is broken when `self.cover` is nil (the default setting).  We need some sort of conditional.